src/test/java/com/epam/rp/ui/tests/LoginTest.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.rp.ui.tests;
+
+import com.epam.rp.utils.ConfigReader;
+import io.qameta.allure.Feature;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Feature("Login")
+public class LoginTest {
+
+    private WebDriver driver;
+    private String defaultUserName;
+    private String defaultUserPassword;
+    private String baseUrl;
+
+    @BeforeEach
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\Denis_Peganov\\drivers\\chromedriver-118.exe");
```

Comment:
I would suggest to use `WebDriverManager` (https://github.com/bonigarcia/webdrivermanager) to simlify configuration.

Reply:
good idea, it's impemented now, thank you